benchmarks/linear_algebra/blas/level1/nrm2/nrm2_generator.cpp

```diff
@@ -0,0 +1,68 @@
+
```

Comment:
Please follow the Tiramisu coding style. This means your code has to look like the rest of Tiramisu code in style. Here no new line is necessary.